velox/exec/tests/HashJoinTest.cpp

```diff
@@ -134,6 +134,19 @@ class HashJoinTest : public HiveConnectorTestBase {
     auto stats = task->taskStats().pipelineStats.front().operatorStats;
     return stats[operatorIndex].inputPositions;
   }
+
+  static uint64_t getOutputPositions(
+      const std::shared_ptr<Task>& task,
+      const std::string& operatorName) {
+    for (const auto& pipelineStat : task->taskStats().pipelineStats) {
+      for (const auto& operatorStat : pipelineStat.operatorStats) {
+        if (operatorStat.operatorType == operatorName) {
+          return operatorStat.outputPositions;
```

Comment:
This works only for single-threaded execution. It would be nice to make this work for multi-threaded execution as well. Also, query plan may have multiple plan nodes that map to the same operator type (e.g. multiple joins). Consider passing in plan node ID to disambiguate.

Reply:
@mbasmanova for multi-threaded execution, is it enough to wait until task competition, or is there something else?

Reply:
It should be sufficient to call waitForTaskCompletion from QueryAssertions.h, which is already being called as part of assertQuery.

Reply:
I added an assert for waitForTaskCompletion